eshop.client/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
this file shouldn't be there - you didn't pay attention carefully when I was doing cleanup

Reply:
you mean whole file index.tsx?
because when I looked up for your video in the end, there was still this file index.tsx

Reply:
update - I delete it and it ruined whole react app so I think its not the thing you mean